_posts/2019-10-21-vscode-quarkus-0.1.0.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: 'Quarkus Tools for Visual Studio Code - 1.1.0 release'
+date: 2019-10-21
+synopsis: New features for Quarkus Tools for Visual Studio Code.
+author: dakwon
+---
+:blank: pass:[ +] 
```

Comment:
What is it suppose to do? I don't see any visual change.

Reply:
The reason for this was to let me insert newlines. I've added newlines before each title like so:
![image](https://user-images.githubusercontent.com/20326645/67715531-67056080-f9a0-11e9-931e-c25af241acc1.png)


If you feel it's not necessary, I will remove it

Reply:
Looks like this without it: 
![image](https://user-images.githubusercontent.com/20326645/67715684-b2b80a00-f9a0-11e9-8ef8-e480aed48fbf.png)